app/views/welcome/_index.html.haml

```diff
@@ -102,6 +102,14 @@
                 = link_to people_path, :class => "morelink explorelink" do
                   = t(:meet)
                   %i.icon-arrow-right
+  #info.homesection
+    .row
+      #ji_left.col-xs-6
+        .pull-right
+          %embed{ src: image_path( "https://static.inaturalist.org/wiki_page_attachments/1248-original.png" ), width: 307 }
```

Comment:
Same here, check it in and give it a more descriptive name than 1248-original.png. Also, FWIW, we generally use SVGs on the home page and in the header to deal with varying display resolutions. Not a requirement, but would get in front of issues like the logos looking pixelated.